Tutorial: How to calculate DAU/MAU ratio (#6045)
PostHog/posthog.com: contents/tutorials/dau-mau-ratio.md

---
title: How to calculate DAU/MAU ratio
date: 2023-05-26
author: ["ian-vanagas"]
showTitle: true
sidebar: Docs
featuredImage: ../images/tutorials/banners/tutorial-14.png
tags: ["insights", "trends"]
---

> The ratio of daily active users over monthly active users, or DAU/MAU ratio, is a popular engagement metric measuring stickiness. It shows what percentage of your users are active users, defining active as having one event captured in your product today.

Reviewer: ```suggestion
The ratio of daily active users over monthly active users, or DAU/MAU ratio, is a popular engagement metric measuring stickiness. It shows what percentage of your users are active and use your product every day.
```

- Snapchat, another popular consumer social app, in [Q1 2023](https://s25.q4cdn.com/442043304/files/doc_financials/2023/q1/Snap-Inc-Q1-2023-Transcript.pdf) had 383 million daily active users and 750 million monthly active users, giving them a DAU/MAU ratio of ~51%.

- MetaMask, a popular crypto wallet, had a DAU/MAU ratio of ~25% in [February 2022](https://twitter.com/ljin18/status/1494804595305959427?lang=en).

- League of Legends, one of the most popular multiplayer online games, has 10.2M daily active users and 153.5M monthly active users in [April 2023](https://activeplayer.io/league-of-legends/), giving it a DAU/MAU ratio of ~6%.

> **Tip:** to benchmark your DAU/MAU ratio, look for the daily and monthly active users of competitors. Companies, especially when public, often report these numbers in financial statements or mention them in blogs, announcements, or interviews.

## The limitations of the DAU/MAU ratio

> Although your DAU/MAU ratio is a great measurement of engagement and stickiness, it is less relevant for products that don’t care about that or not expecting users to engage daily. These include:

Reviewer: ```suggestion
Although your DAU/MAU ratio is a great measurement of engagement and stickiness, it is less relevant for products that don’t care about that or are not expecting users to engage daily. These include:
```

- **B2B SaaS:** many B2B SaaS users have no choice but to use the product. Their employer chooses a product and they come along for the ride. The DAU/MAU ratio doesn’t provide valuable insights. Instead, these companies care more about adoption, retention, feature usage, and net promoter score. Read "[The most useful B2B SaaS product metrics](/blog/b2b-saas-product-metrics)" for more.

> - **Ecommerce:** transactions happen less frequently than daily, meaning DAU/MAU ratio will be low. Daily transactions are not expected here, who would order books from Amazon or clothes from H&M daily? These companies care more about conversion, order value, lifetime value, and customer acquisition costs.

Reviewer: ```suggestion
- **Ecommerce:** transactions happen less frequently than daily, meaning DAU/MAU ratio will be low. Daily transactions are not expected here – few people order books from Amazon or clothes from H&M daily. These companies care more about conversion, order value, lifetime value, and customer acquisition costs.
```

- **Travel:** like ecommerce, people travel infrequently. For example, Airbnb users only book a couple of trips per year. Even apps like Uber care more about weekly active users, as they don’t expect users to ride or order daily. These companies care about similar metrics to commerce companies, like conversion, lifetime value, and customer acquisition cost.

## How to calculate your DAU/MAU ratio

> In PostHog, create an insight with two series, one for "all events" aggregated by "unique users" and another for "all events" aggregated by "monthly active users," then use enable formula mode, and use the formula `A/B`. Use a trend or a number chart to visualize with the Y-axis unit set to "Percent (0-1)."

Reviewer: nit: hyperlink "create an insight"
